test/unit/vl-core.test.html

```diff
@@ -143,6 +143,14 @@
       test('wanneer het spacer-none attribuut aanwezig is zal de correcte class toegevoegd worden', () => {
         assertSpacerClass(fixture('vl-element-fixture'));
       });
+
+      test('de internationalisatie teksten kunnen aangepast worden', () => {
+        const element = fixture('vl-element-fixture');
+        assert.equal(element.getTranslation('select.search_placeholder_value'), 'Zoek item');
```

Comment:
Wat misschien ook nog zou kunnen, maar dan in de component zelf, die keys als constanten definiëren.

Reply:
Ik zie net in vl-upload dat die keys voorlopig intern blijven en je overal een functie hebt voorzien, dus dan is het niet nodig.